src/anchorage/anchorage.service.ts

```diff
@@ -91,6 +91,7 @@ export class AnchorageGraphQLService extends GraphQLService {
             from
             contractId_
             timestamp_
+            timestamp_initiated: timestamp_
```

Comment:
would suggest we add a comment here, nobody most likely would remember why we have this.